scrapli_community/cisco/asa/cisco_asa.py

```diff
@@ -0,0 +1,60 @@
+"""scrapli_community.cisco.asa.cisco_asa"""
+
+from scrapli.driver.network.base_driver import PrivilegeLevel
+from scrapli_community.cisco.asa.async_driver import default_async_on_close, default_async_on_open
+from scrapli_community.cisco.asa.sync_driver import default_sync_on_close, default_sync_on_open
+
+DEFAULT_PRIVILEGE_LEVELS = {
+    "exec": (
+        PrivilegeLevel(
+            pattern=r"^[\S]{1,63}>(\s|)$",
```

Comment:
`(\s|)$"` <- guessing this is to match maybe space after the `>`? If so, can we re-write -> `\s?$` -- had to pop into regex101 to figure out the empty or thing 🤯

Reply:
No worries, have fixed this as well.